Bottom: 8d4edd879d1446e5111e1a6e1c8eed3cf6556cc7
Top:    0c45170fee95402a096abb656e5415727075579c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-11 22:47:18 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
